src/components/RightSidebar/Participants/Participant.spec.js

```diff
@@ -713,7 +713,7 @@ describe('Participant.vue', () => {
 				conversation.participantType = PARTICIPANT.TYPE.MODERATOR
```

Comment:
should adjust the `test()` string to avoid confusion

Reply:
typo ☝🏽